Documentation/operations/system_requirements.rst

```diff
@@ -98,6 +98,16 @@ RancherOS_                 >= 1.5.5
           Linux distribution that works well, please let us know by opening a
           GitHub issue or by creating a pull request that updates this guide.
 
+.. note:: Systemd 245 and above (``systemctl --version``) overrides ``rp_filter`` setting
+          of Cilium network interfaces. This introduces connectivity issues (see
+          `GH-10645 <https://github.com/cilium/cilium/issues/10645>`_ for details). To
+          avoid that, the following can be run:
```

Comment:
nit, better to avoid passive tense. Rather than "the following can be run", "run the following".

```suggestion
          `GH-10645 <https://github.com/cilium/cilium/issues/10645>`_ for details). To
          avoid that, configure rp_filter in systemd using the following commands:
```